tests/test.tracking.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+var tracking = require('tracking');
+
+
+describe('tracking uninitialized', function() {
+
+  beforeEach(function() {
+    this.oldTrackingEnabled = tracking.enabled;
+    this.oldTrackingInitialized = tracking.initialized;
+    tracking.enabled = true;
+    tracking.initialized = false;
```

Comment:
When testing objects that have state I think it's pretty crucial to create a new instance to avoid leakage. Is there a specific reason that the Tracking object is a singleton? I can't see any reason. You could change the class so that it's not a singleton. Then you could start the test with a fresh instance like this:

````
var tracking = rewire('tracking');
var Tracking = tracking.__get__('Tracking');  // get the global Tracking class

beforeEach(function() {
  this.tracking = new Tracking();
});
````